DQM/HcalTasks/BuildFile.xml

```diff
@@ -1,6 +1,9 @@
 <use name="FWCore/Framework"/>
 <use name="rootmath"/>
 <use name="DQM/HcalCommon"/>
+<use name="onnxruntime"/>
```

Comment:
Do we need this `onnxruntime` when we use the next line, `PhysicsTools/ONNXRuntime` ?

Reply:
Do we actually need these additional deps in thie Buildfile.xml ? I do not see any direct usage of onnxruntime or cppunit in DQM/HcalTasks/interface or DQM/HcalTasks/src directory

Reply:
looks like you only need `PhysicsTools/ONNXRuntime` in `DQM/HcalTasks/plugins/BuildFile.xml`. No need to add `onnxruntime` (which comes indirectly from `PhysicsTools/ONNXRuntime` ) and `cppunit` (clean up the unused CPPUNIT commented out code)